engine/language-server/src/main/scala/org/enso/languageserver/search/SuggestionsHandler.scala

```diff
@@ -434,6 +455,30 @@ final class SuggestionsHandler(
           state.suggestionLoadingComplete()
         )
       )
+
+    case JsonSessionTerminated(_) =>
+      logger.info("Processing JsonSessionTerminated")
```

Comment:
```suggestion
      logger.trace("Processing JsonSessionTerminated")
```

Feels more like `trace` or `debug` kind of thing